### PR TITLE
Release 5.7.2

### PR DESCRIPTION
**Pull request by nsingh-branch**
- Added additional fields to initialization and event requests

### CODE REVIEW

File: Branch-SDK/src/main/java/io/branch/referral/ServerRequestLogout.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
+package io.branch.referral;
```
Comment (gdeluna-branch) [inline Branch-SDK/src/main/java/io/branch/referral/ServerRequestLogout.java:1]: Pretty sure this file was removed cc @JagadeeshKaricherla-branch

File: Branch-SDK-TestBed/src/main/java/io/branch/branchandroidtestbed/SettingsActivity.java [hunk 2]
```diff
@@ -35,40 +35,21 @@ void setupRetryEditText() {
 
         retryEditText.setText(Integer.toString(currentRetries));
 
-        retryEditText.setOnEditorActionListener((textView, i, keyEvent) -> {
```
Comment (gdeluna-branch) [inline Branch-SDK-TestBed/src/main/java/io/branch/branchandroidtestbed/SettingsActivity.java:38]: Is this undoing your changes to set custom endpoints?

Comment (nsingh-branch) [reply Branch-SDK-TestBed/src/main/java/io/branch/branchandroidtestbed/SettingsActivity.java:38]: Oh yeah it actually looks like this PR reverts some of the PR instead

File: Branch-SDK/src/main/java/io/branch/referral/ServerRequestLogout.java (hunk 1 above)
Comment (JagadeeshKaricherla-branch) [reply Branch-SDK/src/main/java/io/branch/referral/ServerRequestLogout.java:1]: Yes here https://github.com/BranchMetrics/android-branch-deep-linking-attribution/pull/1133/files

Comment (JagadeeshKaricherla-branch) [reply Branch-SDK/src/main/java/io/branch/referral/ServerRequestLogout.java:1]: ok makes sense since this is of non-master branch.

**Pull request closed without merging**